alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+let answer = "";
+let tallestBuilding = Math.max(...streetArray);
+let numberOfBuildings = streetArray.length;
+let tallestBuildingSoFar = 0;
+let bestArea = tallestBuilding * numberOfBuildings;
+let bannerOne = [tallestBuilding, numberOfBuildings];
+let bannerTwo = [tallestBuilding, numberOfBuildings];
+let bestBannerOne = [tallestBuilding, numberOfBuildings];
+let bestBannerTwo = [tallestBuilding, numberOfBuildings];

```

Comment:
I would rather these broken out into their own individual variables

Reply:
Can you elucidate?  You mean like, bannerOneTallestBuilding, bannerOneNumberOfBuildings, etc., so eight variables instead of four arrays?

Reply:
`bannerOneHeight`/`width` would be appropriate I think

Reply:
Oh, okay.  I can do that.  You're really gonna be unhappy that I added a new element to the array for area.  I'll variableifiy all those then.

Reply:
usually I use objects to keep related data together, like

```js
const banner1 = {tallest: tallestBuilding, numberOfBuildings: numberOfBuildings}
const banner2 = {tallest: tallestBuilding, numberOfBuildings: numberOfBuildings}
```

Reply:
Nuts.  Who should I try to please?  Sensei Dan, or Sensei Eric?

Reply:
If you want to use an object for it I'd say thats fine, I just wouldn't use an array